jest.setup.ts

```
@@ -15,3 +16,8 @@ global.console = {
     // warn: jest.fn(),
     // error: jest.fn(),
 }
+
+// Mock next/cache
+jest.mock('next/cache', () => ({
+    unstable_cache: (fn: Function) => fn,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)